projects/js-packages/components/components/threat-modal/styles.module.scss

```diff
@@ -10,6 +10,24 @@
 	gap: calc( var( --spacing-base ) * 2 ); // 16px
 }
 
+.section .section__toggle {
```

Comment:
Nit - Is the specificity necessary here? One of the benefits of the BEM style class naming is the ability to keep specificity flat (i.e. only one selector deep).